packages/grafana-data/src/field/fieldState.ts

```diff
@@ -101,7 +101,16 @@ export function decoupleHideFromState(frames: DataFrame[], fieldConfig: FieldCon
       };
 
       // original with perm overrides
-      field.config.custom.hideFrom = hideFrom;
+      if (field.config.custom !== undefined) {
```

Comment:
I only added this because the tests were hitting it, if youd rather have me leave it and change the test, I can do that!